internal/config/plugin_installer.go

```diff
@@ -687,7 +692,13 @@ func PluginCommand(out io.Writer, cmd string, args []string) {
 		plugins := GetInstalledVersions(false)
 		fmt.Fprintln(out, "The following plugins are currently installed:")
 		for _, p := range plugins {
-			fmt.Fprintf(out, "%s (%s)\n", p.Pack().Name, p.Version)
+			if p.Pack().Name == "initlua" {
+				fmt.Fprintf(out, "initlua\n")
```

Comment:
nit: just to keep it in line with the below printfs, we could `fmt.Fprintf(out, "%s\n", p.Pack().Name)`